src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const inverse = {};
```

Comment:
Sound like a function, let's change the naming for 
```suggestion
  const inversedRobot = {};
```